apartmenttherapy/oxygen-lib-ui
# Shannon/cloudinary dpr

#### What does this PR do & why?
- change images using cloudinary to use dpr
   - dpr automatically selects the proper image size per browser dpi
- remove dpi calculations from resolution list for cloudinary fetched images

#### How can this be manually tested?
- also use pr: https://github.com/apartmenttherapy/oxygen/pull/575
- desktop browser
  - navigate to http://localhost:8080/collection/bundle-samples
  - test in safari and chrome at different browser sizes (refresh per resize because dpr image is delivered on page load)
  - inspect any image
  - hover over the `img > src` and see the dimensions of image delivered is the close to the image displayed per media query for chrome and around 2.5x the size for safari on retina screens and around 1x on non retina screens.
- iphone
  - connect iphone to computer via phone usb
  - open up network utility
  - under info tab see `IP Address`
  - type this address into your iphone's safari + `:[localhost port number]/` + the rest of the url above
  - open safari on your desktop
  - under the Develop tab select iphone > safari the url that you went to on your phone (should be the ip address)
  - drill down the dom to an image and see that the image dimensions delivered is around 2.5x the size of the image displayed

#### Any questions or concerns about this change?

_(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)_

#### Deployment Notes
also has a oxygen pr: https://github.com/apartmenttherapy/oxygen/pull/575

_(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)_

#### Link to story card and other documentation
https://app.asana.com/0/543787180217047/801498898904057

#### GIF

#### Have you remembered to...?
